docs/src/man/basics.md

```diff
@@ -0,0 +1,1220 @@
+    2 │     1     22  female      2  own      little           moderate
+    3 │     2     49  male        1  own      little           NA
+    4 │     3     45  male        2  free     little           little
+    5 │     4     53  male        2  free     little           little          ⋯
+    6 │     5     35  male        1  free     NA               NA
+    7 │     6     53  male        2  own      quite rich       NA
+    8 │     7     35  male        3  rent     little           moderate
+  ⋮   │   ⋮      ⋮      ⋮       ⋮       ⋮            ⋮                ⋮        ⋱
+  994 │   993     30  male        3  own      little           little          ⋯
+  995 │   994     50  male        2  own      NA               NA
+  996 │   995     31  female      1  own      little           NA
+  997 │   996     40  male        3  own      little           little
+  998 │   997     38  male        2  own      little           NA              ⋯
+  999 │   998     23  male        2  free     little           little
+ 1000 │   999     27  male        2  own      moderate         moderate
+                                                  4 columns and 985 rows omitted
+```
+
+You can see that Julia representation (unlike python pandas) displays the data type of the column,
+whether it is a `string`, `int` or `float`.
+
+To access the columns directly (i.e. without copying) you can use `german.col`, `german."col"`,
+`german[!, :col]` or `german[!, "col"]`. The two latter syntaxes are more flexible as they allow
+us passing a variable holding the name of the column, and not only a literal name. Columns name
+can be either symbols (written as `:col`, `:var"col"` or `Symbol("col")`) or strings (written as `"col"`).
+Variable interpolation into the string using `$` does not work if you have forms like `german."col"`
+and `:var"col"`. You can also access the column using an integer index specifying their position.
+
+Since `german[!, :col]` does not make a copy, changing the elements of the column vector returned by this
+sysntax will affect the values stored in the original `german`. To get a **copy** of the column you can use
+`german[:, :col]`: changing the vector returned by this syntax does not change `german`.
+
+```jldoctest dataframe
+julia> german.Sex
+1000-element PooledArrays.PooledVector{String, UInt32, Vector{UInt32}}:
+ "male"
+ "female"
+ "male"
+ "male"
+ "male"
+ "male"
+ "male"
+ "male"
+ "male"
+ "male"
+ ⋮
+ "male"
+ "male"
+ "male"
+ "male"
+ "female"
+ "male"
+ "male"
+ "male"
+ "male"
+
+julia> german.Sex === german[!, :Sex]
+true
+
+julia> german.Housing === german[:, 5]
+false
+```
+
+You can obtain the column names as strings using the `names` function:
+```jldoctest dataframe
+julia> names(german)
+10-element Vector{String}:
+ "id"
+ "Age"
+ "Sex"
+ "Job"
+ "Housing"
+ "Saving accounts"
+ "Checking account"
+ "Credit amount"
+ "Duration"
+ "Purpose"
+```
+
+ you can also get column names with a given `eltype`:
```

Comment:
maybe be more verbose, writing `eltype` is a shorthand we use because we communicete to people who know what it means.

Reply:
I am not getting this. Sorry. What exactly we want to change?

Reply:
if it is clear for you what `eltype` means then we can leave it.